amit/index.js

```diff
@@ -0,0 +1,59 @@
+let apples = 20;
+let oranges = 30;
+let equal = apples == oranges;
+console.log(equal); //false
+
+let equal2 = apples != oranges;
+console.log(equal2); // true
+
+let equal3 = apples > oranges;
+console.log(equal3);
+
+let equal4 = apples <= oranges;
+console.log(equal4);
+
+let equal5 = oranges > apples;
+console.log(equal5);
+
+// que.6
+let mangoes = 5;
+let mul = mangoes * apples;
+let add = mangoes + oranges;
+let check = mul > add;
+console.log(check);
+
+//que.7
+let sub = apples - mangoes;
+let div = oranges / mangoes;
+let check2 = sub < div;
+console.log(check2);
+
+//que 8
+let allEqual = (mangoes == apples) == oranges;
+console.log(allEqual);
+
+// que 9. Check whether the remainder of apples divided by mangoes and the remainder of oranges divided by mangoes are equal.
```

Comment:
The solution of 9th exercise would be like this:
console.log((apples % mangoes) == (oranges % mangoes));